File: tms-webapi-master/TMS.Common/ViewModels/SendMailModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TMS.Common.ViewModels
{
    public class SendMailModel
    {
        public string[] toEmail { get; set; }
        public string[] ccMail { get; set; }
        public string Title { get; set; }
        public string CreateBy { get; set; }
        public string Action { get; set; }
        public string EmailType { get; set; }
        public string EmailSubject { get; set; }
        public string ReasonReject { get; set; }
        public string Category { get; set; }

        //Request
        public string GroupName { get; set; }
        public string RequestTypeName { get; set; }
        public string DetailReason { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        //OT Request
        public string OTDate { get; set; }
        public string OTDateType { get; set; }
        public string OTTimeType { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        //Abnormal
        public string ExplanationDate { get; set; }
        public string ExplanationReason { get; set; }
        public string Actual { get; set; }
        public string Description { get; set; }
        public string Sender { get; set; }
        public string Receiver { get; set; }
    }
}

File: tms-angular2-master/src/app/core/models/sendMailModel.ts
export class SendMailModel {
    public toEmail: string[];
    public Title: string;
    public Action:string;
    public CreateBy: string;
    public EmailType: string;
    public EmailSubject:string;
    public ReasonReject:string;
    public Category:string;
    
    constructor(Title: string, CreateBy: string,toEmail:string[],Action:string,EmailType:string,EmailSubject:string,ReasonReject:string,Category:string) {
        this.Title = Title;
        this.CreateBy = CreateBy;
        this.toEmail = toEmail;
        this.Action = Action;
        this.EmailType = EmailType;
        this.EmailSubject = EmailSubject;
        this.ReasonReject = ReasonReject;
        this.Category = Category;
    }
}
